user_guide_src/source/concepts/factories.rst

```diff
@@ -109,11 +107,29 @@ file might look like this::
 This would prevent conflict of an unrelated third-party module which happened to have an
 unrelated "Filters" path in its namespace.
 
-Runtime configuration is even easier: simply supply the desired configuration values to
-``Factories`` using the ``setConfig()`` method and they will be merged with the default
-values and stored for the next call::
+setOptions Method
+-----------------
+
+The ``Factories`` class has a static method to allow runtime option configuration: simply
+supply the desired array of options using the ``setConfig()`` method and they will be
```

Comment:
renamed to `setOptions()` ?

Reply:
Good catch! I did a global search-and-replace but it must have applied to `.php` files only.